cmd/utils/tss_options.go

```diff
@@ -33,7 +33,7 @@ func ErrorHandlerJitterChannelBufferSizeOption(configKey *int) *config.ConfigOpt
 		Usage:       "Set the buffer size of the Error Handler Jitter channel.",
 		OptType:     types.Int,
 		ConfigKey:   configKey,
-		FlagDefault: 100,
+		FlagDefault: 1000,
```

Comment:
Is there a way in which we determined these values? Or are they just a random default value right now?